target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -47,6 +47,9 @@ ath79_setup_interfaces()
 	mikrotik,routerboard-map-2nd)
 		ucidef_set_interface_lan "eth0 eth1"
 		;;
+	mikrotik,routerboard-960pgs)
+		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4 lan5" "sfp lan1"
```

Comment:
Maybe move lan1 to lan group by default?

Reply:
Port labeling is bit misleading see https://cdn.mikrotik.com/web-assets/rb_images/1220_xl.webp
They are labeled as 1 to 5 where port 1 and sfp ports are wan ports.